teal.modules.general.Rproj

```diff
@@ -1,4 +1,5 @@
 Version: 1.0
+ProjectId: 003390c8-e31f-4b87-957d-7943778020cc
```

Comment:
New Rstudio version :) I wonder what this is used for, as it was not disclosed on the release notes